pyatlan/client/atlan.py

```diff
@@ -2023,13 +2050,15 @@ def get_lineage_list(
         else:
             assets = []
             has_more = False
+        aggregations = self.get_aggregations(raw_json)
```

Comment:
There are no aggregations via the lineage list endpoint, they should only be used via index search.